tests/flow/test_msf.py

```
@@ -101,7 +101,7 @@ def test_invalid_invocation(self):
             try:
                 self.graph.query(q)
                 self.env.assertFalse(True)
-            except redis.exceptions.ResponseError as e:
+            except redis.ResponseError as e:
```

Ruff (0.15.7)

[error] `redis` may be undefined, or defined from star imports

(F405)